### DISCUSSION TITLE
[로또 step1] 셀렉터 상수화

### DISCUSSION THREAD

**Discussion by al-bur** [woowacourse-study/2022-code-review-study-2#2] [General]
[#82 - comment](https://github.com/woowacourse/javascript-lotto/pull/82#discussion_r813875417)

리뷰어님이 말대로 많은 크루분들이 셀렉터를 상수화하는 것을 보았습니다.

셀렉터의 상수화에 대해서 의견 나눠보면 좋을 것 같습니다

**Comment by euijinkk** [comment]
- 안
다른 군데에서 여러개 써야한다면 설계가 잘못된 것이다. 안 쓰는 것에 동의 한다.

- 콜라
동의
- 앨버
동의

상수화 해주면 또 들어가서 봐야한다.
